pom.xml

```diff
@@ -77,8 +82,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.3</version>
 				<configuration>
-					<source>1.7</source>
-					<target>1.7</target>
+					<source>1.8</source>
```

Comment:
also why I tend to define `<maven.compiler.target/>` and `<maven.compiler.source/>` in properties to not redefine the complete `build >> plugins >> plugin >> configuration`.

Reply:
Yeah, I think there are still many things improvable like these indeed. I am happy to make another pass once this is merged to see where we can make things more current/clean. For instance, I'd like to remove all the currently commented out code (like the `// log...` calls + the various `System.out.println` calls, which should be either removed, or made to be displayed by adding e.g. a new `-verbose` mode in the `Options` class).